shared/src/test/scala/squants/space/VolumeSpec.scala

```diff
@@ -167,7 +167,7 @@ class VolumeSpec extends AnyFlatSpec with Matchers {
   }
 
   it should "return Length when cube rooted" in {
-    CubicMeters(27).cubeRoot should be(Meters(3))
+    CubicMeters(64).cubeRoot should be(Meters(4))
```

Comment:
Easy fix

Reply:
Ideally you could choose the number type to have exact calculations but squants only supports doubles